defaults/main.yml

```diff
@@ -14,6 +14,10 @@ k8s_redis_enabled: false
 k8s_redis_version: "5.0.6"
 k8s_redis_volume_size: "20Gi"
 
+k8s_elasticsearch_enabled: false
+k8s_elasticsearch_version: "6.5.0"
```

Comment:
I wonder if this config would also work with the latest 6.x release, which appears to be 6.8.6?

https://hub.docker.com/_/elasticsearch?tab=description